homeassistant/components/sensor/linux_battery.py

```diff
@@ -35,45 +35,57 @@
 ATTR_VOLTAGE_MIN_DESIGN = 'voltage_min_design'
 ATTR_VOLTAGE_NOW = 'voltage_now'
 
+ATTR_HEALTH = 'health'
+ATTR_PRESENT = 'present'
+ATTR_STATUS = 'status'
+
 CONF_BATTERY = 'battery'
+CONF_DISTRIBUTION = 'distribution'
```

Comment:
I'm not really happy with this name. For me "distributions" are different flavors of an OS and not different OSes. Why not `system` (`operating_system` seems a bit long)?

Reply:
Android is based on a modified Linux kernel, so in my opinion, is a special distribution of Linux, just like Fedora, or Debian, or Arch, is not another operating system.